src/groups/Sentence.jsx

```diff
@@ -29,147 +30,73 @@ class Sentence extends React.Component {
   }
   getStyles() {
     return {
-      base: {
-        fontSize: 24
+      container: {
+
       }
     };
   }
-  makeTotalCommentsPhrase() {
-    const statsCommentMinTotalDefault = 0;
-    const statsCommentMaxTotalDefault = 10000;
-    /* if the min and max for comments ARE NOT the defaults, render. */
-    if (
-      this.props['stats.comments.total'].userMin !== statsCommentMinTotalDefault ||
-      this.props['stats.comments.total'].userMax !== statsCommentMaxTotalDefault
-    ) {
-      return (`
-        wrote more than
-        ${this.props['stats.comments.total'].userMin}
-        but less than
-        ${this.props['stats.comments.total'].userMax}
-        total comments,
-      `);
-    }
-  }
-  makeAcceptRatioPhrase() {
-    const acceptRatioMinDefault = 0;
-    const acceptRatioMaxDefault = 1;
-    /* if the min and max for comments ARE NOT the defaults, render. */
-    if (
-      this.props['stats.accept_ratio'].userMin !== acceptRatioMinDefault ||
-      this.props['stats.accept_ratio'].userMax !== acceptRatioMaxDefault
-    ) {
-      return (`
-        had a more than
-        ${this.props['stats.accept_ratio'].userMin}
-        but less than
-        ${this.props['stats.accept_ratio'].userMax}
-        accept ratio for comments they submitted,
-      `);
+
+  isDefault(filterName) {
+    let isDefault = true;
+    const f = this.props[filterName];
+    const maxDifferent = f.userMax !== f.max && f.userMax < f.max;
+    const minDifferent = f.userMin !== f.min;
+    if (maxDifferent || minDifferent) {
+      isDefault = false;
     }
+    return isDefault;
   }
-  makeRepliesPerCommentPhrase() {
-    const repliesPerCommentMaxDefault = 0;
-    const repliesPerCommentMinDefault = 1;
-    /* if the min and max for comments ARE NOT the defaults, render. */
-    if (
-      this.props['stats.replies_per_comment'].userMin !== repliesPerCommentMaxDefault ||
-      this.props['stats.replies_per_comment'].userMax !== repliesPerCommentMinDefault
-    ) {
-      return (`
-        between
-        ${this.props['stats.replies_per_comment'].userMin}
-        and
-        ${this.props['stats.replies_per_comment'].userMax}
-        replies per comment on average,
-      `);
-    }
+
+  formatName(name) {
```

Comment:
could I have a comment here? I think this function converts to title case, but I'm not sure. Or just rename the function `toTitleCase`